pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -361,7 +362,35 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			ObjectMeta: kapi.ObjectMeta{
 				Name: JobControllerRoleName,
 			},
-			Rules: []authorizationapi.PolicyRule{},
+			Rules: []authorizationapi.PolicyRule{
+				// JobController.jobController.ListWatch
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("jobs"),
+				},
+				// JobController.syncJob() -> updateJobStatus()
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("update"),
+					Resources: sets.NewString("jobs/status"),
+				},
+				// JobController.podController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("pods"),
+				},
+				// JobController.podControl (RealPodControl)
+				{
+					Verbs:     sets.NewString("create", "delete"),
```

Comment:
Exactly that. Jobs are very similar to rc controller, which also does not get pods. Rather just watch changes and each sync does a List using the selector provided.